src/WebUI/Components/ServiceComponent.scss

```diff
@@ -10,4 +10,24 @@
 
 .s-de-heading {
     padding-left: 20px;
-}
\ No newline at end of file
+}
+
+.s-pod-list-content {

```

Comment:
.s-pod-list-content {
    display: flex;
    flex-direction: column;
    justify-content: space-around;
    margin-top: 10px; 
}

.s-pod-col-header {
    color: $secondary-text;


if these are common settings we should set them in common place